src/interfaces/packagingSObjects.ts

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export namespace PackagingSObjects {
```

Comment:
SObject schema for tooling objects are not yet available via jsforce2, adding here for the time being.